schema.go

```diff
@@ -340,16 +341,16 @@ func (indexField *IndexField) DecodeMsgpack(d *msgpack.Decoder) error {
 	return errors.New("unexpected schema format (index fields)")
 }
 
-func (conn *Connection) loadSchema() (err error) {
+func GetSchema(conn *Connection) (*Schema, error) {
```

Comment:
I don't see a reason to return a pointer.
```suggestion
func GetSchema(conn *Connection) (Schema, error) {
```

Reply:
Updated.